tests/scripts/thread-cert/node.py

```diff
@@ -3265,6 +3265,9 @@ def __init__(self, nodeid, name=None, **kwargs):
         self.nodeid = nodeid
         self.name = name or ('Host%d' % nodeid)
         super().__init__(nodeid, **kwargs)
+        self.bash('service otbr-agent stop')
+        self.bash('service mdns stop')
+        self.bash('service avahi-daemon stop')
```

Comment:
move to `start` ?

Reply:
I'm inclined to put them here otherwise otbr-agent will keep running until `start` is explicitly called. HostNode is usually the last node runs `start`.

Reply:
Make sense to me. Thanks.